### ISSUE TITLE
[Bug] Event Gate Labels

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#4445]
### Short Description of the issue:
Event Gate labels only show when the player is also controlling a Spaceship Computer. For example hotkeys are not shown on the HUD when controlling a turret, hovercraft or just a yoke with no Computer.

Also, the Default HUD label is an empty field in the Event Gate, not sure if that's supposed to show anything
![image](https://user-images.githubusercontent.com/27857711/147974328-947d8230-5dd3-4b33-9edb-d8a1a1b3f6c2.png)

### Steps to reproduce the issue:
- Link a Yoke to an Event Gate
- Set the Event Gate to Display on HUD
- Enter the Yoke - No Hotkey is shown

### Environment
- Starship EVO version: 22w01a
- Graphics Card: RX 580 8GB

**Comment by tsunamayo** [comment]
@ProPeach Hi, is you event gate linked to some weapons? Thx

**Comment by ProPeach** [comment]
No, it was only linked to an indicator light to verify that the event gate was working as a hotkey.

To better show the bug, here the hotkey is shown on the HUD when the Yoke is also connected to a Spaceship Core
![image](https://user-images.githubusercontent.com/27857711/147993713-4d177e2a-dd88-406e-a82f-eb566905766e.png)

If I link the same Yoke to a Hover computer instead, or a turret computer, or no computer the hotkey isn't shown on the HUD
![image](https://user-images.githubusercontent.com/27857711/147993771-4ae6fc60-d251-477e-aedc-25b8174023ce.png)

If this is only supposed to work with weapons, it'd be great to have it work without too so you can also see hotkeys on the HUD for moving parts like doors or landing gear, wings etc
